src/robot/output/output.py

```diff
@@ -22,6 +22,7 @@
 
 
 class Output(AbstractLogger):
+    _registered_loggers = []
```

Comment:
Is there a reason to have `_registered_logger` as a class attribute? Is there a need to share a list of loggers? If not, this definitely should be an instance attribute.